pkgs/by-name/pa/papers/package.nix

```diff
@@ -99,14 +97,17 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   mesonFlags = [
-    "-Dnautilus=false"
```

Comment:
Hmm, looks like our Darwin CI does not like Nautilus. Could you make it optional on `!stdenv.isDarwin`?

Reply:
Fixed